services/docker-compose.yml

```diff
@@ -236,6 +238,41 @@ services:
       - default
       - interactive_services_subnet
 
+  webserver-garbage-collector:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/webserver:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
+    environment:
+      <<: *webserver-environment
+      WEBSERVER_GARBAGE_COLLECTOR: '{"GARBAGE_COLLECTOR_INTERVAL_S": 30}'
+    env_file:
+      - ../.env
+      - ../.env-webserver-garbage-collector
+
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "python3",
+          "/home/scu/services/web/server/docker/healthcheck.py",
+          "http://localhost:8080/v0/"
+        ]
+      interval: 30s
+      timeout: 120s
+      retries: 3
+      start_period: 30s
```

Comment:
we should probably also review these, in prod the start period is pretty long and usually the services are up in less than 10s